src/mapnode.h

```diff
@@ -250,7 +250,7 @@ struct MapNode
 	/*
 		Gets list of collision boxes
 	*/
-	std::vector<aabb3f> getCollisionBoxes(INodeDefManager *nodemgr) const;
+	std::vector<aabb3f> getCollisionBoxes(INodeDefManager *nodemgr, u8 neighbors = 0) const;
```

Comment:
can you update your patch to change the proto like this in this PR: void getCollisionBoxes(INodeDefManager *nodemgr, std::vector<aabb3f>& boxes, u8 neighbors = 0) . Same for getNodeBoxes functions.
Those objects are very large to copy in memory